plugins/poll/assets/javascripts/discourse/components/modal/poll-ui-builder.hbs

```diff
@@ -99,7 +105,7 @@
       </div>
     {{/unless}}
 
-    {{#unless this.isRegular}}
+    {{#unless this.dontShowForIrvOrRegular}}
```

Comment:
Can we use a different name? Two negations is a bit hard to read "unless" + "dont..."

Reply:
totally agree, improved this and also for one other instance in same Component: https://github.com/discourse/discourse/pull/27155/commits/cf413f254e87ea7420a229bfde341f4183ccfcc8